### PR TITLE
Correlation-based analysis of our data

### PR DESCRIPTION
**Pull request by joao-binenbojm**

### CODE REVIEW

File: corremap.m~ [hunk 1]
```diff
@@ -0,0 +1,15 @@
+function corrmap(data,sel,unit,type)
+    % [] = RASTER_PLOT(data,sel,opt,type)
```
Comment (rr1017) [inline corremap.m~]: Change name to CORRMAP and change input variable names

File: corremap.m~ [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+    % data - given struct array 
+    % sel - struct with fields:
+        %.trial - range or single trial value
+        %.angle - range or single angle value
+        %.unit - range or single unit value
+    % unit - integerargument:
```
Comment (rr1017) [inline corremap.m~]: Add space

File: corremap.m~ [hunk 3]
```diff
@@ -0,0 +1,15 @@
+function corrmap(data,sel,unit,type)
+    % [] = RASTER_PLOT(data,sel,opt,type)
+    % data - given struct array
```
Comment (rr1017) [inline corremap.m~]: What does this function contain? It's just comments

File: corrmap.m~ [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+    % [] = CORRMAP(data,sel,show)
+    % data - given struct array 
+    % sel - struct with fields:
+        %.trial - range or single trial value
+        %.angle - single angle value
+        %.unit - single unit value
+    % show - boolean argument: 
+        % true - computes cross-correlation heatmap between neuronal unit
+        % and other units
+        % false - does not compute/disply heatmap
```
Comment (rr1017) [inline corrmap.m~]: Display

File: xcorr_similarity.m [hunk 5]
```diff
@@ -0,0 +1,33 @@
+function sim_mat = xcorr_similarity(data)
+% Computes cross-correlation dissimilarity for every neuron unit pair
```
Comment (rr1017) [inline xcorr_similarity.m]: Modify format, sim_mat = XCORR_SIMILARITY(data).

File: xcorrmap.m [hunk 6]
```diff
@@ -0,0 +1,50 @@
+function [map, mean_spikes] = xcorrmap(data,sel,show)
```
Comment (rr1017) [inline xcorrmap.m:1]: Why are there three xcorrmap??? Which one is the good one? hahaha

**Review by rr1017 — CHANGES_REQUESTED** [review]
Just minor stuff. Big issue is why are there three xcorrmap?

File: corremap.m~ (hunk 1 above)
Comment (joao-binenbojm) [reply corremap.m~]: Old version of the function was still lingering even though I thought it was deleted. My bad.

File: corremap.m~ (hunk 3 above)
Comment (joao-binenbojm) [reply corremap.m~]: Deleted, old version

File: xcorrmap.m (hunk 6 above)
Comment (joao-binenbojm) [reply xcorrmap.m:1]: this is hahaha

**Pull request merged**